redis-lettuce/src/main/java/io/micronaut/configuration/lettuce/cache/RedisCache.java

```diff
@@ -207,6 +215,46 @@ public String getName() {
         return value;
     }
 
+    public <K> Map<K, Object> get(Collection<K> keys) {
+      return get(keys, Argument.of(Object.class));
+    }
+
+    public <K, T> Map<K, T> get(Collection<K> keys, Argument<T> requiredType) {
+        Map<byte[], K> serializedKeyMap = new HashMap<>(keys.size());
```

Comment:
Again does order matter here? Otherwise maybe a `LinkedHashMap`

Reply:
Only used locally for matching the serialized key back to the original (since some key serialization is one-way, we can't deserialize). So order doesn't matter.